molecule/cp-kafka-plain-rhel/molecule.yml

```diff
@@ -6,6 +6,7 @@ platforms:
       - kafka_broker
       - zookeeper
     image: geerlingguy/docker-centos7-ansible
+    dockerfile: ../Dockerfile.j2
```

Comment:
Why are we adding a docker file to this?  I believe this scenario was originally designed to run without a docker file to test for that scenario.

Reply:
I see. Has this been successful earlier? This fails withe error 
```
"Could not find the requested service confluent-zookeeper: host"
```
I could verify there isn't any package installed on vanilla image.

Reply:
Got resolved. This works without dockerfile. 
Thanks for pointing this out.